### DISCUSSION TITLE
background-image doesn't seem to load when placed in css file, but works when in inline style

### DISCUSSION THREAD

**Discussion by anjanesh** [vercel/next.js#59456] [Help]
### Summary

background-image doesn't seem to load when placed in css file, but works when in inline style - this is only on production - on local, there isn't this issue. I've hosted this on vercel.

### Additional information

```js
.header_wrapper
{
    background-image: url(../public/background/banner.jpeg);
}
```
This doesn't work :
```
<div className={styles.header_wrapper}>
```
But this does :
```
<div className={styles.header_wrapper} style={{ "backgroundImage": "url('/background/banner.jpeg')"}}>
```
```


### Example

_No response_

**Comment by 99lalo** [comment]
Its not a class issue? Try using a color
If its a path issue make sure that the relative path its correct, if you're using vscode you can ctrl-click the path and it should take you to the image.

**Reply by anjanesh** [reply]
_if you're using vscode you can ctrl-click the path and it should take you to the image._
It does take me to the image in VSCode. But on production (vercel.com) it doesn't load at all.

**Reply by Barbagidon** [reply]
did u solve this problem?

**Reply by Barbagidon** [reply]
i have the same

**Reply by anjanesh** [reply]
I couldn't solve this problem in vercel hosting - I just offloaded the image to a CDN and linked it to it in CSS.

**Comment by abojoud2006** [comment]
if you using tailwind => bg-[url('../public/images/Logo-Icon.svg')]
if you using css file => background-image: url("../public/images/Logo-Icon.svg");
-----------------
its works for me
